src/llama.cpp

```diff
@@ -57,6 +57,12 @@
     #include <io.h>
 #endif
 
+#if __cplusplus >= 202000L
+    #define LU8(x) (const char*)(u8##x)
```

Comment:
you should use a proper cast, instead of a c-style one.

Reply:
Why? It's not like there are any "proper" casts in the file itself? All other casts are C-style.